osu.Game.Rulesets.Osu/Skinning/Legacy/LegacySpinner.cs

```diff
@@ -127,5 +129,33 @@ protected override void Dispose(bool isDisposing)
             if (DrawableSpinner != null)
                 DrawableSpinner.ApplyCustomUpdateState -= UpdateStateTransforms;
         }
+
+        /// <summary>
+        /// A <see cref="Container"/> simulating osu!stable's absolute screen-space,
+        /// for perfect placements of legacy spinner components with legacy coordinates.
+        /// </summary>
+        protected class LegacyCoordinatesContainer : Container
```

Comment:
don't see the point of splitting this out at all. it's just a container. it was inline already, you've just moved it out somewhere else but made it a class for some reason. it's not reusable either because the class is nested and protected.

Reply:
actually never mind, it's reused, but still I don't get why it has to be done this way instead of overriding `Content` or something

Reply:
because the legacy bonus counter pieces are outside of the `LegacySpinner` component, if it's better to move the bonus counter inside `LegacySpinner`, then this can be inlined.

Reply:
Sort of shoddy to be justifying these changes with a pull further down the chain. Makes this one unreviewable on its own.

I'm also not sure why the bonus counter wouldn't be inside of the spinner. It's part of it after all, isn't it?